consul-secure-acl-vault/index.json

```diff
@@ -0,0 +1,79 @@
+{
+    "pathwayTitle": "Generate Consul Tokens with HashiCorp Vault",
+    "icon": "fa-consul",
+    "title": "Generate Consul Tokens with HashiCorp Vault",
+    "description": "Use Vault's Consul secrets engine to generate Consul ACL tokens",
+    "details": {
+        "steps": [
+            {
+                "title": "Provisioning Lab Infrastructure",
```

Comment:
```suggestion
                "title": "Lab Infrastructure Provisioning",
```

Reply:
I want to move the verb to the end to make it even more clear that the student's don't need to do anything for this step.